ui/Dockerfile

```diff
@@ -1,27 +1,35 @@
-FROM node:9 as builder
```

Comment:
Why was the node 9 changed to 8?

Reply:
Hi @JaDogg please look in to [this](#375) we have issues with nodesass and higher versions of node.

Reply:
@kmehant node-sass release notes state that it works with node version between 6 & 7. Is version 8 working fine?

Reply:
@VibhorCodecianGupta yes, version 8 is working fine.